### PR TITLE
KVIrc certfp config

### PR DESCRIPTION
**Pull request by darsie**
Added KVIrc certfp config

### CODE REVIEW

**Comment by darsie** [comment]
Lots of new territory there for me, I hope I did it well. I guess the formatting can be improved.

File: content/_guides/certfp.md [hunk 1]
```diff
@@ -159,6 +159,28 @@ to `CREATE NEW` and when you tap this, a certificate will be randomly generated
 and a certificate fingerprint will be displayed. Tap the tick symbol on the top
 right of the screen to save.
 
+### KVIrc
+
+Create `libera.pem` e.g. in your KVIrc config directory
+(e.g. `~/.config/KVIrc`), or move it there.
+
+In `Settings/Configure KVIrc...` expand `Connections` and click `Advanced`.
```
Comment (meldra) [inline content/_guides/certfp.md:167]:
```suggestion
In `Settings/Configure KVIrc...` expand `Connection` and click `Advanced`.
```

File: content/_guides/certfp.md [hunk 2]
```diff
@@ -159,6 +159,28 @@ to `CREATE NEW` and when you tap this, a certificate will be randomly generated
...
+
+Check `Use SSL private key`.
+
+At `Private key location` configure the location of libera.pem.
+
+If you have certificate and private key in separate files, use these respectively.
+
+Click `OK` or `Apply`.
+
+
```
Comment (meldra) [inline content/_guides/certfp.md:183]:
```suggestion
```

**Comment by meldra** [comment]
there's a better one so we'll go with that.

**Pull request closed without merging**